.github/workflows/test_and_build_image.yaml

```diff
@@ -37,6 +37,12 @@ jobs:
           key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
           restore-keys: |
             ${{ runner.os }}-go-
+      - name: Install yq
+        run: |
+          sudo rm /bin/yq
+          wget https://github.com/mikefarah/yq/releases/download/v4.28.1/yq_linux_amd64.tar.gz
```

Comment:
Why are we installing 4.28.1 ?

Reply:
> Why are we installing 4.28.1 ?

To fix the `yq` issue [here](https://github.com/k8ssandra/k8ssandra-operator/issues/820).

Reply:
I just tried with version 4.30.6, and it works. So maybe we can update to that version, but 4.30.8 seems to have the issue in #820

Reply:
So it seems the issue exists with version 4.30.7 and 4.30.8, but not 4.30.6

Reply:
It might make sense to issue a bug report, since that 820 was referring to a bug in 4.9.8, not 4.30. And I don't see any change like that between 4.30.6 and 4.30.8 release notes.

Reply:
Or, alternatively.. it's this we hit actually: https://github.com/mikefarah/yq/issues/1515

In that case, we should probably fix our implementation and not change yq version ;)